INTRODUCTION.md

```diff
@@ -245,8 +246,8 @@ configuration property.
 
 Compression is performed on the batch of messages in the local queue, the
 larger the batch the higher likelyhood of a higher compression ratio.
-The local batch queue size is controlled through the `batch.num.messages` and
-`linger.ms` configuration properties as described in the
+The local batch queue size is controlled through the `batch.num.messages`,
+`batch.size`, and `linger.ms` configuration properties as described in the
```

Comment:
batch.max.bytes is less ambiguous and doesn't require a trip to the docs to understand its purpose.